app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -34,17 +34,17 @@ def shared_update
         user_id: current_user.id,
         threatmetrix_session_id: idv_session.threatmetrix_session_id,
         request_ip: request.remote_ip,
-        double_address_verification: capture_secondary_id_enabled,
+        double_address_verification: double_address_verification,
       )
 
       return true
     end
 
     private
 
-    def capture_secondary_id_enabled
-      current_user.establishing_in_person_enrollment&.
-          capture_secondary_id_enabled || false
+    def double_address_verification
+      # if in person return true else return false
+      analytics_arguments[:analytics_id] == 'In Person Proofing'
```

Comment:
temporarily using `double_address_verification` to check if we're in the ipp flow in files related to the proofing job and result adjudication. Will add a ticket that adds flag for enrollments that are created in the ipp flow in order to distinguish those enrollments in the files that still reference `double_address_verification`